New Posts (#38)
Vanilla-OS/handbook: _posts/2023-01-11-install-vpn-from-file.md

---
title: Installing an OpenVPN File Without Using Terminal
description: This guide provides a step-by-step process for installing an OpenVPN file without using the terminal. It is designed to help users who may be uncomfortable with command-line interfaces or who simply prefer a more user-friendly approach.
date: 2023-01-11
layout: article
authors: 

>     - [Jacob Gonzales](https://kutt.it/jacobgonzales)

Reviewer: Nice work, I will review this soon. The authors field only supports GitHub usernames currently.

Author: > Nice work, I will review this soon. The authors field only supports GitHub usernames currently.

oh okay, my github username will do! I plan to add more guides!

Reviewer: > I plan to add more guides catered toward non-tech-savvy people

Feel free to let us know about the guides you are writing in the #docs-channel in our Discord server (Link in README) so that we can track these articles [here](https://github.com/orgs/Vanilla-OS/projects/2).

Author: > > I plan to add more guides catered toward non-tech-savvy people
> 
> Feel free to let us know about the guides you are writing in the #docs-channel in our Discord server (Link in README) so that we can track these articles [here](https://github.com/orgs/Vanilla-OS/projects/2).

I would definitely, if it wasn't discord. Would you like me to make a issue before making new guides so you can add them to the documentation progress?

Reviewer: > I would definitely, if it wasn't discord... Would you like me to make a issue before making new guides so you can add them to the documentation progress?

Yeah, that would do, @mirkobrombin can you add this person to the docs team?

For this, I have created [this](https://github.com/Vanilla-OS/handbook/issues/40), you can add "Closes #40" to this PRs Description. In future, once you are part of the team you can directly add entries to the project and create issues.

Author: > 

great! by adding directly do you mean, I'll be able to push post directly to the main directory of this repo?

Reviewer: > > 
> 
> great! by adding directly do you mean, I'll be able to push post directly to the main directory of this repo?

No, You can access the project with todo list and progresd. Being part of the docs team you can start  discussions internally since you don't prefer Discord.

Reviewer: You can discuss here after joining

![image](https://user-images.githubusercontent.com/26346867/211982042-e1c5dae6-87f6-45f2-a92a-33f6ea0ea2e1.png)

Currently there are 0 discussion 😅

> ---
> title: Installing an OpenVPN File Without Using Terminal
> description: This guide provides a step-by-step process for installing an OpenVPN file without using the terminal. It is designed to help users who may be uncomfortable with command-line interfaces or who simply prefer a more user-friendly approach.
> date: 2023-01-11
> layout: article
> authors: 
>     - [Jacob Gonzales](https://kutt.it/jacobgonzales)
> published: true
> ---
> 
> # Step-by-Step Guide: 
> 
> 1. Press the Windows key (super key)
> 2. Search for "vpn"
> ![Photo Example](/assets/uploads/install-openvpn-from-file-search.webp)
> 3. Select the result shown next to the settings icon, called "Network"
> 4. This will bring you to the "Network" interface
> 5. Click the plus button next to the VPN icon
> ![Photo Example](/assets/uploads/install-openvpn-from-file-vpn-add.webp)
> 6. Three options will be displayed; select "Import from file..."
> 7. This will open your file explorer 
> 8. Select the .ovpn config file downloaded from your VPN provider
> 9. The OpenVPN profile will now be installed. 
> 
> `Note: You may be prompted to enter login credentials after installing the OpenVPN configuration file.`

Reviewer: ```suggestion
---
title: Installing an OpenVPN File Without Using Terminal
description: Learn how to install an OpenVPN File graphically
date: 2023-01-11
layout: article
authors: 
    - gonzoknows
published: true
---

## Introduction

This guide provides a step-by-step process for installing an OpenVPN file graphically without using the terminal. It helps users who may be uncomfortable with command-line interfaces prefer a more user-friendly approach.

## Navigating to Network Settings

- Press the Windows/Super key.
- Search for "VPN" or "vpn" in the search.

![Search for vpn](/assets/uploads/install-openvpn-from-file-search.webp)

- Select "Network" from the Settings search result.
-  This will bring you to the "Network" panel interface.
	- Alternatively, you can directly navigate to the "Network" panel by opening Settings from the Control Center in the top right corner.
	
## Importing the OpenVPN file

- Once you have navigated to the Network panel, Click on the plus (`+`) button beside the VPN.

![Plus Button beside VPN](/assets/uploads/install-openvpn-from-file-vpn-add.webp)

- Various options will be displayed in the menu, then select "**Import from file...**"
- This will open a Files dialogue box to select the file to import.
- Select the **`.ovpn`** config file downloaded from your VPN provider.
- This will install the OpenVPN profile on your computer. 

_**Note**_: You may be prompted to enter login credentials after installing the OpenVPN configuration file.

## Conclusion

- Now, you can successfully connect to the OpenVPN profile seamlessly.
```

Reviewer: Updated the file, and fixed issues with rendering after this the PR is GTG. In future, maybe try testing the page locally (more info in the README).

Reviewer: > > Updated the file, and fixed issues with rendering after this the PR is GTG. In future, maybe try testing the page locally (more info in the README).
> 
> ahah yea #41

Noticed it

---
title: Installing an OpenVPN File Without Using Terminal
description: Learn how to install an OpenVPN File graphically
date: 2023-01-11
layout: article
authors: 
    - gonzoknows
published: true
---

## Introduction

This guide provides a step-by-step process for installing an OpenVPN file graphically without using the terminal. It helps users who may be uncomfortable with command-line interfaces prefer a more user-friendly approach.

## Navigating to Network Settings

- Press the Windows/Super key.
- Search for "VPN" or "vpn" in the search.

![Search for vpn](/assets/uploads/install-openvpn-from-file-search.webp)

- Select "Network" from the Settings search result.
-  This will bring you to the "Network" panel interface.
	- Alternatively, you can directly navigate to the "Network" panel by opening Settings from the Control Center in the top right corner.
	
## Importing the OpenVPN file

> - Once you have navigated to the Network panel, Click on the plus (`+`) button beside the VPN.

Reviewer 2: ```suggestion
- Once you have navigated to the Network panel, click on the plus (`+`) button beside the VPN.
```

![Plus Button beside VPN](/assets/uploads/install-openvpn-from-file-vpn-add.webp)

> - Various options will be displayed in the menu, then select "**Import from file...**"

Reviewer 2: ```suggestion
- Various options will be displayed in the menu, select "**Import from file...**"
```
